src/lib/y2storage/filesystems/base.rb

```diff
@@ -37,6 +37,12 @@ class Base < Mountable
       #   @return [Array<Filesystems::Base>] all the filesystems in the given devicegraph
       storage_class_forward :all, as: "Filesystems::Base"
 
+      # @!method self.in_network?
```

Comment:
This was the first method that I created I just copied the lines above... My mistake...